### ISSUE TITLE
Interview Checklist

### ISSUE THREAD

**Issue by marlitas** [phetsims/center-and-variability#235]
Here will go a checklist of all the items @catherinecarter expects to be completed by interview time. We can then go through this list and prioritize based on the remaining time.

**Comment by matthew-blackman** [comment]
The devs agreed to do a self test to get a preliminary idea of where the sim stands for interviews on 6/5. These are some of the features that seem reasonable. @catherinecarter feel free to modify or add to this list!

General
- [ ] Soccer ball stacking and z-order are correct
- [ ] Soccer ball dragging is correct
- [ ] Median highlight accurate and updating at the correct time
- [ ] Mean highlight position is accurate
- [ ] Number lines in all contexts lined up to within a reasonable margin
- [ ] 'Reset all' and 'Clear data' buttons are working correctly
- [ ] All data is visible in the accordion box plots for all values of 'maxKicks'
- [ ] Accordion box and info dialog components are readable and do not occlude one another
- [ ] Sonification in place for kick/landing and card sorting

Median Screen
- [ ] Card sorting and 'sort data' feature in accordion box working correctly
- [ ] 'Predict median' tool and 'median' highlight working correctly
- [ ] 'Reset all' and 'Clear data' buttons behave correctly for all screen-specific elements

Mean & Median Screen
- [ ] 'Predict' tools and 'median' highlight working correctly
- [ ] Data point highlight animation in accordion box working correctly
- [ ] 'Reset all' and 'Clear data' buttons behave correctly for all screen-specific elements

Variability Screen
- [ ] Individual players have correct kick distributions
- [ ] Players can be investigated in parallel to one another (scenes working correctly)
- [ ] 'Interval Tool' working correctly
- [ ] Data visualizations and metric are accurate in accordion box plots
- [ ] 'Reset all' and 'Clear data' buttons behave correctly for all screen-specific elements

**Comment by catherinecarter** [comment]
Not sure what these "Soccer ball dragging is correct" means. Could one of you clarify what this means?

**Comment by marlitas** [comment]
Self Dev Test Results (more to come maybe...):

- [ ] Accordion Box does not reset to open when the reset button is pressed.

**Comment by marlitas** [comment]
> Not sure what these "Soccer ball dragging is correct" means. Could one of you clarify what this means?

@catherinecarter, This is in reference to making sure that interactions with dragging the ball aren't buggy. The ball's position updates correctly, as does any connected data. Animations are terminated properly, etc.

**Comment by marlitas** [comment]
The interview is currently happening, so this checklist is no longer needed. I opened side issues for any bugs or questions found here.
